kratos/geometries/geometry.h

```diff
@@ -1375,7 +1375,7 @@ class Geometry : public PointerVector<TPointType>
         for ( unsigned int pnt = 0; pnt < this->IntegrationPointsNumber( ThisMethod ); pnt++ )
         {
             this->Jacobian( J, pnt, ThisMethod);
-            rResult[pnt] = MathUtils<double>::Det(J);
```

Comment:
@RiccardoRossi , if you look to my previous comment and to the code, the GeneralizedDet does a check, if the Matrix is square it computes the standard determinant